evm/deploy-test-gateway-token.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const { ethers } = require('hardhat');
+const {
+    Contract,
+    Wallet,
+    providers: { JsonRpcProvider },
+    utils: { parseEther, keccak256, defaultAbiCoder, arrayify, hexlify, hexZeroPad },
+    constants: { AddressZero, MaxUint256 },
+} = ethers;
+const { Command, Option } = require('commander');
+
+const { mainProcessor, getContractJSON } = require('./utils');
+const { addExtendedOptions } = require('./cli-utils');
+
+async function getCommandId(gateway) {
```

Comment:
Is this approach desirable? In your tests, how long does it take to find the next not-executed command id?